ledger_app.toml

```diff
@@ -0,0 +1,7 @@
+[app]
+build_directory = "./"
+sdk = "C"
+devices = ["nanos", "nanox", "nanos+", "stax"]
+
```

Comment:
Could be interesting to differentiate plugin and app ?

Reply:
How so?

Reply:
No difference for compilation and tests. I'm just wondering if it could be relevant to have this info
 for future usage including some specific features dedicated to the plugins (for example in Vscode extension)

Reply:
It could be added when the need arises then, the format is simple enough to allow such evolution without any issue (that or any other feature).
If we add data while not knowing how we're going to use them, I fear we could regret it. Like we did with the legacy manifest, where it turned out that having the `rust` qualifier as a key was a strong limitation for other usages.

Reply:
ok 👍